src/components/toolbar/AppUserMenu.tsx

```diff
@@ -37,6 +38,10 @@ const AppUserMenu: FC<AppUserMenuProps> = ({
     const { user } = useApi();
     const { data: isQuickConnectEnabled } = useQuickConnectEnabled();
 
+    const onDownloadManagerClick = useCallback(() => {
+        shell.openDownloadManager();
+        onMenuClose();
+    }, [ onMenuClose ]);
     const onClientSettingsClick = useCallback(() => {
```

Comment:
```suggestion
    }, [ onMenuClose ]);

    const onClientSettingsClick = useCallback(() => {
```
To follow the style.